modules/openxAnalyticsAdapter.js

```diff
@@ -138,10 +165,10 @@ function getPublisherPlatformId() {
 }
 
 function getPublisherAccountId() {
-  if (initOptions.publisherAccountId !== undefined) {
-    if (typeof initOptions.publisherAccountId === 'number') {
-      if (initOptions.publisherAccountId > -1) {
-        return initOptions.publisherAccountId;
+  if (analyticsConfig.publisherAccountId !== undefined) {
+    if (typeof analyticsConfig.publisherAccountId === 'number') {
+      if (analyticsConfig.publisherAccountId > -1) {
+        return analyticsConfig.publisherAccountId;
```

Comment:
something i've been pondering about, why is this a number, but the adunit ID is a string?

should we just make this a string also?

Reply:
I'll update these type checks in the future.  This is copied over from roxot code.